helpers/for.js

```diff
@@ -3,30 +3,37 @@
 const _ = require('lodash');
 
 const factory = () => {
-    return function(from, to, context) {
-        const options = arguments[arguments.length - 1];
+    return function(from, to, context, options) {
         const maxIterations = 100;
-        var output = '';
+        let output = '';
 
         function isOptions(obj) {
             return _.isObject(obj) && obj.fn;
         }
 
         if (isOptions(to)) {
+            options = to;
             context = {};
             to = from;
             from = 1;
 
         } else if (isOptions(context)) {
+            options = context;
             if (_.isObject(to)) {
                 context = to;
                 to = from;
                 from = 1;
             }
         }
 
-        if (to <= from) {
-            return;
+        if (options.hash['zeroDiffStart'] !== true) {
```

Comment:
Let's change the option name to `includeLast` which I think will make more sense to people.